ex01.ex

```diff
@@ -86,14 +86,20 @@ defmodule Ex01 do
   # function. The examples below will make this clearer :)               #
   ########################################################################
 
-  create_adder = your_anonymous_function(1)
 
-  add_2  = create_adder.(2)
-  add_99 = create_adder.(99)
+  defmodule Adder do
```

Comment:
## -5

This was supposed to be an anonymous function

~~~ elixir
  create_adder = fn (n1) -> 
    fn (n2) ->
       n1 + n2 
    end 
  end
~~~